packages/mock-block-dock/src/bottom-view.tsx

```diff
@@ -0,0 +1,133 @@
+// @todo rename
```

Comment:
👀

Reply:
😅 Missed this. Although I'm not sure if `bottom-view` is the best name for this. Do you have a better name I can call it?

Reply:
`footer`? 🤔

Reply:
`devtools`? (not a fan of `view`)

Reply:
Honestly not sure why I didn't think of devtools. Will update to that in a follow up PR